pymc3/smc/smc.py

```diff
@@ -398,34 +407,48 @@ def __init__(
             observed data
         function: python function
             data simulator
+        params: list
+            names of the variables parameterizing the simulator.
         model: PyMC3 model
         var_info: dict
             generated by ``SMC.initialize_population``
-        distance: str
-            Distance function. Available options are ``absolute_error`` (default) and
-            ``sum_of_squared_distance``.
-        sum_stat: bool
-            Whether to use or not a summary statistics.
+        distance : str or callable
+            Distance function. The only available option is ``gaussian_kernel``
+        sum_stat: str or callable
+            Summary statistics. Available options are ``indentity``, ``sorted``, ``mean``,
+            ``median``. The user can pass any valid Python function
```

Comment:
Same comment as above, regarding compatibility…
You can also use `(…, function, *, params, ...)` to require everything behind `,* , ` as a kwarg.

Reply:
I will change it. I doubt the ABC sampler is widely used. But I have plans to change that in the short-term :-)